README.md

```diff
@@ -277,6 +277,7 @@ Thanks goes to these wonderful people ([emoji key][emojis]):
 ### Tutorials
 #### Tutorial rpearce
 * [Ramda Chops: Function Currying](https://robertwpearce.com/blog/ramda-chops-function-currying.html)
+* [Safer JavaScript with the Maybe Type](https://egghead.io/courses/safer-javascript-with-the-maybe-type)
```

Comment:
Awesome. there is something else going on here tho.
we need to add the video to `avanslaars` section in the contributors section. This is done by running the following in the project

`npm run setup` 

then
`npm run contributors:add avanslaars video`

That will add the badge and the link. then we need to make a section
```md
#### Video avanslaars
* [Safer JavaScript with the Maybe Type](https://egghead.io/courses/safer-javascript-with-the-maybe-type)
```

Reply:
Also, looks like the TOC needs to be updated as well.
So after those changes are made, run `npm run doctoc`

And another also, if you would like you can add yourself to the contributors for your updates to the docs, just run: `npm run contributors:add zeucxb doc`